dashboarddoc.md

```diff
@@ -0,0 +1,156 @@
+---
+
+User Dashboard Interaction
+==========================
+{: .anchor}
+
+This document illustrates how a Solar Forecast Arbiter user may use the
+Dashboard to complete the following actions:
+
+-   [Create a new Site](#create-a-site).
+
+-   [Create a new Observation or Forecast](#create-an-observation-or-forecast).
+
+-   [Upload Data to an Observation or Forecast](#upload-data).
+
+
+Create a Site
+-------------
+{: .anchor}
+
+1. Navigate to sites listing page using ‘Sites’ link in the left
+   sidebar. At the top of the Site listing click ‘Create new Site’.
+<img class="my-3" src="/images/SitesListing.png"/>
+
+2. Enter the metadata for your Site. Selecting a site type of 
+   'Power Plant' will prompt you for additional fields. 
+
+   <img class="my-3" src="/images/TooltipDetail.png" class="figure"/>
+   <div class="figure mb-3">
+   Descriptive tooltips are displayed when a user selects a field. 
+   </div>
+
+   - *Weather station site creation form*
+   <img class="my-3" src="/images/SiteForm.png"/>
+
+   - *Power plant site creation form*
+   <img class="my-3" src="/images/plant_site_form.png"/>
+
+3. After submission, you will be redirected to a Site
+   page which displays the new Site’s metadata and allows you
+   to create associated Observations and Forecasts. See
+   [Create an Observation or Forecast](#create-an-observation-or-forecast).
+
+   *Power Plant Site Page*
+   <img class="my-3" src="/images/plant_site.png"/>
+
+
+
+Create an Observation or Forecast
+---------------------------------
+{: .anchor}
+
+To create an Observation or Forecast, an associated site must already exist.
+See [Create a Site](#create-a-site) for instructions.
+
+### Create an Observation
+{: anchor} 
+
+1.  Navigate to the Site listing page using the ‘Sites’ link in the left
+	sidebar. Select the site for which you are adding an Observation.
+
+2.  Click the “Create new Observation” button on the Site page.
+  <img class="my-3" src="/images/weather_site.png"/>
+
+3.  Enter metadata for your Observation. On submission, you will be redirected
+    to an Observation page which displays the new Observation metadata and a
+    link to add Observation data. See [Upload Data](#upload-observation-data).
+- *Observation form*
+<img class="my-3" src="/images/weather_obs.png"/>
+- *Created Observation page*
+<img class="my-3" src="/images/observation.png"/>
+
+### Create a Forecast
+{: anchor}
+
+1.  Navigate to the Site listing page using the ‘Sites’ link in the left
+	sidebar. Select the site for which you are adding a Forecast.
+
+2.   Click the “Create new Forecast” button on the Site page.
+  <img class="my-3" src="/images/plant_site.png"/>
+
+3.  Enter metadata for your Observation. On submission, you will be redirected
+    to an Observation page which displays the new Forecast metadata and a
+    link to add Forecast data. See [Upload Data](#upload-forecast-data).
+- *Forecast form*
+<img class="my-3" src="/images/forecast_form.png"/>
```

Comment:
this png has a few of the old parameter names (intervals per submission --> run length / issue frequency, interval duration --> interval length, interval label should show beginning instead of start)